src/zentables/zentables.py

```diff
@@ -819,14 +827,101 @@ def _swap_column_levels(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def _combine_n_pct(
-    df_n: pd.DataFrame, df_pct: pd.DataFrame, digits: int = 1
+    df_n: pd.DataFrame,
+    df_pct: pd.DataFrame,
+    digits: int = 1,
+    suppress_symbol: str = "*",
 ) -> pd.DataFrame:
     """
     Helper function that formats and combines n and pct values
     """
     df_n_str = df_n.astype(str)
     df_pct_str = df_pct.applymap(lambda x: f" ({x:.{digits}%})", na_action="ignore")
-    return df_n_str.add(df_pct_str)
+    return df_n_str.add(df_pct_str).fillna(f"{suppress_symbol}")
+
+
+def _is_between(x, ceiling: int = 10):
+    """
+    Helper function that verifies if the input is below ceiling value
+    """
+    return x in range(1, ceiling)
+
+
+def _add_random_noise(x, rng):
+    """
+    Helper function that adds random noise to entry according a Uniform distribution
+    """
+    return x + rng.uniform(0, 1)
```

Comment:
Not sure why this needs to be a function. Also, if you want to break ties b/w integer values, you probably want the upper limit to be at most 0.5